ddterm/shell/extension.js

```diff
@@ -140,6 +140,28 @@ class ExtensionDBusInterface {
     }
 }
 
+class DesktopEntry {
+    static file = Gio.File.new_for_path(
+        GLib.build_filenamev([GLib.get_user_data_dir(), 'applications', `${APP_ID}.desktop`])
+    )
```

Comment:
Could you make everything non-static, create the instance in `enable()`, and set the reference to `null` in `disable()`? Or convert `file()` to a function/property?

https://gjs.guide/extensions/review-guidelines/review-guidelines.html#destroy-all-objects

Reply:
Of course! Thanks for link to the docs :)